jest/fileTransformer.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
+
+ const path = require('path');
+
+ module.exports = {
+  process(src, filename, _config, _options) {
+    return 'module.exports = ' + JSON.stringify(path.basename(filename)) + ';';
+  }
+};
```

Comment:
Is this like, fake Webpack-file-loader? :P

Reply:
Yeah, it just used as a replacement for the modules as mapped in https://github.com/buildkite/frontend/pull/745/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R193